packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -261,6 +266,9 @@ const getMetaStyles = (theme: GrafanaTheme2) => ({
     margin: theme.spacing(0.5, 0, 0),
     lineHeight: theme.typography.bodySmall.lineHeight,
     overflowWrap: 'anywhere',
+  }),
+  metadataItem: css({
+    // Needed to allow for clickable children in metadata
     zIndex: 0,
```

Comment:
```suggestion
    position: 'relative',
```
also works, dunno which is easier to understand